features/google/src/model/importers/utils/lang-constants.js

```diff
@@ -0,0 +1,31 @@
+export const langConstants = {
+    AccessLogImporter: {
```

Comment:
I'm trying to think about an alternative that wouldn't need either to make selective imports (and thus instruct the user to just deal with every section in a different file) or to avoid this kind of boilerplate, which is not even consistent (it's translated in some case, it's not in others).
Only thing I can think of is to put all language strings in a single regex. In these cases, `$[\w ]*[aA][ck]tivit[\w ]+` seems to fit all of them.
Also, this implies that either we know the language in advance (an educated guess would be using the same as is detected by `silly-i18n`) or we loop through all versions.
If you have considered or want to consider any of these solutions, please feel free. Otherwise, this one fits the bill and is Good Enough (TM)